src/components/cc-orga-member-card/cc-orga-member-card.js

```diff
@@ -82,22 +87,6 @@ export class CcOrgaMemberCard extends LitElement {
       isCurrentUser: false,
     };
 
-    /**
-     * used to:
-     * - wrap buttons when the component width is below 740.
-     * - switch to a vertical card layout when the width is below 580.
-     * - show buttons and badges below each other when the width is below 350.
-     *
-     * @protected
-     */
-    this.breakpoints = {
-      width: [BREAKPOINT_TINY, BREAKPOINT_SMALL, BREAKPOINT_MEDIUM],
-    };
```

Comment:
using the `lit-labs` controller API, we no longer need to declare a public "protected" `breakpoints` property.

The controller will trigger the trigger the `willUpdate` lifecycle hook everytime a resize happens.